packages/core/src/lib/implementation/execute-plugin.ts

```diff
@@ -117,39 +117,39 @@ export async function executePlugins(
   plugins: PluginConfig[],
   options?: { progress: boolean },
 ): Promise<PluginReport[]> {
-  const { progress = false } = options || {};
+  const { progress = false } = options ?? {};
 
-  const progressName = 'Run Plugins';
-  const progressBar = progress ? getProgressBar(progressName) : null;
-
-  const pluginsResult = await plugins.reduce(async (acc, pluginCfg) => {
-    const outputs = await acc;
+  const progressBar = progress ? getProgressBar('Run plugins') : null;
 
+  const pluginsResult = plugins.map(pluginCfg => {
     progressBar?.updateTitle(`Executing  ${chalk.bold(pluginCfg.title)}`);
 
     try {
-      const pluginReport = await executePlugin(pluginCfg);
+      const pluginReport = executePlugin(pluginCfg);
       progressBar?.incrementInSteps(plugins.length);
-      return outputs.concat(Promise.resolve(pluginReport));
-    } catch (e: unknown) {
+      return Promise.resolve(pluginReport);
+    } catch (error: unknown) {
       progressBar?.incrementInSteps(plugins.length);
-      return outputs.concat(
-        Promise.reject(e instanceof Error ? e.message : String(e)),
+      return Promise.reject(
+        error instanceof Error ? error.message : String(error),
       );
     }
-  }, Promise.resolve([] as Promise<PluginReport>[]));
+  });
```

Comment:
Revert these changes, please :pray: We execute the plugins sequentially on purpose, executing them in parallel can affect the results of some plugins (e.g. Lighthouse measurements).